### PR TITLE
Prac 05 Code Review Request

### PR DESCRIPTION
**Pull request by jessepurcell**
Hi @luke185 
Could you please review my code. Please take a close look at my variable and function naming, Thanks.

### CODE REVIEW

File: prac_05/emails.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+"""
+
+
+def main():
+    """A program to store and print emails and names"""
+    user_to_email = {}
+    user_name, email = get_email()
+    while email != "":
+        choice = input(f"Is your name {user_name}? (Y/n) ")
+        if choice not in "Yy ":
```
Comment (luke185) [inline prac_05/emails.py:14]: A comment explaining the string choice here might be good

File: prac_05/hex_colors.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+"""
+A program that shows the hex value of colors
+"""
+
+COLOR_TO_HEX = {"Absolute Zero": "#0048ba", "Acid Green": "#b0bf1a", "Alice Blue": "#f0f8ff",
+                "Alizarin Crimson": "#e32636", "Amaranth": "#e52b50", "Amber": "#ffbf00",
+                "Aqua": "#00ffff", "Baker Miller Pink": "#ff91af", "Bittersweet Shimmer": "#bf4f51",
+                "British Racing Green": "#004225"}
+
```
Comment (luke185) [inline prac_05/hex_colors.py:9]: Great code! 👍

File: prac_05/state_names.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+                "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+# print(CODE_TO_NAME)
+
+for code in CODE_TO_NAME:
```
Comment (luke185) [inline prac_05/state_names.py]: Would using ` for code, name in CODE_TO_NAME.items(): ` be more efficient?

File: prac_05/wimbledon.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+
+def print_champions(champions_to_wins):
+    """Prints the wimbledon champions"""
+    print("Wimbledon Champions:")
+    for champion, wins in champions_to_wins.items():
+        print(f"{champion} {wins}")
+
+
+def get_wimbledon_data():
```
Comment (luke185) [inline prac_05/wimbledon.py:39]: Would these function names be suitable for other programs?

**Review by luke185 — COMMENTED** [review]
Great work, only a few minor possible improvements 👍

File: prac_05/state_names.py (hunk 3 above)
Comment (jessepurcell) [reply prac_05/state_names.py]: I don't believe it's more effecient but it's certainly more readable. Changed.

Comment (jessepurcell) [reply prac_05/state_names.py]: I'll retract my statement as running the file through pylint suggested changing it to that as well...

**Pull request merged**